scss/overrides_slate/_bootswatch.scss

```diff
@@ -28,7 +28,7 @@ $web-font-path: 'https://fonts.googleapis.com/css?family=Open+Sans:400,700' !def
 }
 
 .btn-outline-primary {
-  color: $white;
+  color: $link-color;
```

Comment:
just curious why you picked $link-color over $primary?

Reply:
Primary is a darker blue and was hard to see in dark mode - dark blue on dark gray. Link color was a better choice for a text-only clickable thing, and the animation went well with the actual primary button color on hover. I suppose this could go sideways if we choose link color to be not blue, for branding or whatever. I don't see that happening, though.

Reply:
ok, just curious!